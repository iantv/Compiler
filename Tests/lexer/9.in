0xAAAAAAAA
0XAAAAAAAA