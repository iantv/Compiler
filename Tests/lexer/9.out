1	1	k		identifier
1	2	++		++
1	4	++		++
1	6	+		+
1	7	--		--
1	9	-		-
1	10	++		++
1	12	--		--
1	14	-		-
1	15	+		+